tests/testthat/test-Model-class.R

```diff
@@ -1094,6 +1094,120 @@ test_that("No NA is returned in dose calculations for OneParExpNormalPrior model
   expect_false(is.na(calc_dose(0.95)))
 })
 
+# OneParExpExpPrior ----
+
+## constructor ----
+
+test_that("OneParExpExpPrior object can be created with user constructor", {
+  result <- expect_silent(
+    OneParExpExpPrior(
+      skel_probs = seq(from = 0.1, to = 0.9, length = 5),
+      dose_grid = 1:5,
+      lambda = 2
+    )
+  )
+  expect_valid(result, "OneParExpExpPrior")
+})
```

Comment:
We should also test if the custom values for class slots are stored correctly. See e.g. `CohortSizeRange` class in `test-Rules-class.R` as a reference.

Reply:
@wwojciech I think I do not fully understand the logic. I tried the code
```
test_that(".OneParExpExpPrior works as expected", {
  result <- expect_silent(.OneParExpExpPrior())
  expect_valid(result, "OneParExpExpPrior")
})
```
But that end in an error, even though the class of result is `OneParExpExpPrior`.

Reply:
See for example this one:
```
test_that("CohortSizeRange object can be created with user constructor", {
  result <- expect_silent(CohortSizeRange(c(0, 30, 50), c(20, 60, 90)))
  expect_valid(result, "CohortSizeRange")
  expect_identical(result@intervals, c(0, 30, 50))
  expect_identical(result@cohort_size, c(20L, 60L, 90L))
})
```

Reply:
added

Reply:
@wwojciech - Thanks. I think I got it now.